.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
```

Comment:
@bcapdevilac shouldn't we run at least the build on travis for now to make sure that works?